client/my-sites/customer-home/cards/features/blogging-prompt/index.jsx

```diff
@@ -18,8 +19,19 @@ import './style.scss';
 const BloggingPromptCard = () => {
 	const dispatch = useDispatch();
 	const translate = useTranslate();
-	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const selectedSiteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const primarySiteId = useSelector( ( state ) => getPrimarySiteId( state ) );
+	let siteId = selectedSiteId;
+
+	// We need a site ID to request prompts
+	// If no selected site ID, fallback to using Primary site ID
+	if ( siteId === null ) {
+		siteId = primarySiteId;
+	}
+
 	const siteSlug = useSelector( ( state ) => getSelectedSiteSlug( state ) );
+	const notificationSettingsLink = '/me/notifications' + siteSlug ? '#' + siteSlug : '';
```

Comment:
When I go back to use this card on My Home, the "Manage Notifications" link doesn't work any more.